src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -260,6 +260,39 @@ def test_aks_create_and_update_with_managed_aad_enable_azure_rbac(self, resource
             self.check('aadProfile.enableAzureRbac', False)
         ])
 
+    @AllowLargeResponse()
+    @AKSCustomResourceGroupPreparer(random_name_length=17, name_prefix='clitest', location='westus2')
+    def test_aks_create_and_update_with_node_restriction(self, resource_group, resource_group_location):
```

Comment:
Queued a [pipeline](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=59472133&view=results) to generate the recording file for this newly added test case.

Reply:
Queued a new [pipeline](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=59726478&view=results), it passed.